backend/src/main/java/com/dbms/georgia_express/controller/CustomerLoginController.java

```diff
@@ -16,17 +18,22 @@ public class CustomerLoginController {
     @Autowired
     private CustomerLoginService customerLoginService;
 
+    @Autowired
+    private JwtTokenUtil jwtTokenUtil;
+
     @PostMapping("/register")
     @Operation(summary = "Registers an user")
-    public ResponseEntity<CustomerLogin> register(@RequestBody RegistrationRequest request) {
+    public ResponseEntity<LoginResponse> register(@RequestBody RegistrationRequest request) {
         CustomerLogin customerLogin = customerLoginService.register(request);
-        return ResponseEntity.ok(customerLogin);
+        String token = jwtTokenUtil.generateToken(customerLogin.getUsername());
+        return ResponseEntity.ok(new LoginResponse(token));
     }
 
     @PostMapping("/login")
     @Operation(summary = "Login a customer")
-    public ResponseEntity<CustomerLogin> login(@RequestBody LoginRequest request) {
+    public ResponseEntity<LoginResponse> login(@RequestBody LoginRequest request) {
         CustomerLogin customerLogin = customerLoginService.login(request);
```

Comment:
Make sure customerLoginService.login also generates a token as other APIs will verify the token.